mysys/my_getexe.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2017, MariaDB Plc
```

Comment:
Is it though?

Reply:
Vlad was the plc author of the entire my_get_exepath function that was copied here and its authored year of 2017 was used.